src/watch/index.ts

```diff
@@ -4,7 +4,10 @@ import { constants as osConstants } from 'node:os';
 import path from 'node:path';
 import { command } from 'cleye';
 import { watch } from 'chokidar';
+import picomatch from 'picomatch';
```

Comment:
Picomatch is actually the largest dependency now:
https://pkg-size.dev/chokidar%20glob-parent%20is-glob%20normalize-path%20picomatch

Is there a lighter alternative?